README.md

```diff
@@ -390,13 +390,21 @@ php artisan discord:interactions:register
 
 #### Testing Webhook Locally
 
-For local development, use tools like ngrok to expose your local server:
+For local development, we recommend using [expose](https://expose.dev/) to expose your local server. Expose is a popular choice among Laravel/PHP developers for creating secure tunnels to localhost.
+
+First, install expose globally:
+
+```shell
+composer global require exposedev/expose
+```
+
+Then expose your local Laravel server:
 
 ```shell
-ngrok http 8000
+expose 8000
```

Comment:
To use expose, run the `expose share` command after starting the Laravel local server.
```
php artisan serve
expose share http://127.0.0.1:8000
```
If you are not using `php artisan serve`, please adjust it to suit your environment.